app/page/authorization/transaction.tsx

```diff
@@ -169,9 +171,22 @@ function UserOperationConfirmation(props: {
       userOp.setPaymasterAndData(
         await paymentOption.paymaster.requestPaymasterAndData(userOp)
       )
-      userOp.setSignature(
-        await sender.sign(userOp.hash(entryPoint, network.chainId))
-      )
+
+      // Sign user operation
+      try {
+        console.log("signing user operation...")
+        setIsSigning(true)
+        const signature = await sender.sign(
+          userOp.hash(entryPoint, network.chainId)
+        )
+        userOp.setSignature(signature)
+        setIsSigning(false)
+      } catch (signErr) {
+        setIsSigning(false)
+        console.log("signErr", signErr)
+        setToast("Verify passkey failed.", "failed")
```

Comment:
nit: I found it is not so ideal to have fixed status as string. Maybe we could refactor it at next PR for two possible ways

(1) Use enum for status, or
(2) Derive `setSuccessToast`, `setSentToast`, `setFailedToast` and use each of them when needed.